static/js/Misc.jsx

```diff
@@ -1349,6 +1349,8 @@ class DisplaySettingsButton extends Component {
   render() {
     let style = this.props.placeholder ? {visibility: "hidden"} : {};
     let icon;
+    const altText = Sefaria._('Text display options')
+    const classes = classNames({readerOptionsTooltip: 1, "tooltip-toggle": true});
```

Comment:
There's no reason to use `classNames` if both class names are always true. Just make it a static string.